cwltool/builder.py

```diff
@@ -282,7 +282,7 @@ def bind_input(self,
                 self.files.append(datum)
                 if (binding and binding.get("loadContents")) or schema.get("loadContents"):
                     with self.fs_access.open(datum["location"], "rb") as f:
-                        datum["contents"] = f.read(CONTENT_LIMIT)
+                        datum["contents"] = f.read(CONTENT_LIMIT).decode("utf-8")
```

Comment:
Hmmm... In the CWL specification we don't specify the encoding of user files..

Reply:
Ah yes, that probably needs to be added